nifi-nar-bundles/nifi-marklogic-bundle/NOTICE.txt

```diff
@@ -0,0 +1,2106 @@
+MarkLogic NiFi Processors
+
+
+Copyright � 2018 MarkLogic Corporation.
+
+This project is licensed under the Apache License, Version 2.0 (the "License"); you may not use this project except in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+
+Please direct questions, comments and requests to fossreview@marklogic.com. 
+
+Open source software required to be made available under license is included herein. In the event you are unable to obtain a copy of such open source software, please contact fossreview@marklogic.com and a copy will be made available to you.
+
+
+The following software may be included in this project (last updated May 1, 2018):
+
+Apache Commons Codec� 1.7 

```

Comment:
Please have this follow formatting similar to what is shown in this example

https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-azure-bundle/nifi-azure-nar/src/main/resources/META-INF/NOTICE#L30

Reply:
Please model all of these dependency references off examples you can find in the existing source tree.  I dont see any new dependencies here that you would have trouble finding examples of how to reference.  If you have questions on how to find any of these please let me know.  Formatting is important to help this be readable/consistent/maintainable.